app/config/config.php

```diff
@@ -1,8 +1,10 @@
 <?php
 
 $db = false;
+
 if (getenv('DATABASE_URL')) {
     $db = parse_url(getenv('DATABASE_URL'));
+    $db = array_map('urldecode', $db);
```

Comment:
Good catch!

Doctrine is using `rawurldecode` for this: https://github.com/doctrine/dbal/blob/v2.8.0/lib/Doctrine/DBAL/DriverManager.php#L262

Reply:
I've changed it to rawurldecode and restructured it a bit.